actions/sequester/README.md

```diff
@@ -23,6 +23,7 @@ To install the GitHub actions:
    - You'll need to add two secret tokens to access the OSPO REST APIs and Quest Azure DevOps APIs.
    - *OSPO_KEY*: Generate a PAT [here](https://ossmsft.visualstudio.com/_usersSettings/tokens). UserProfile: Read is the only access needed.
    - *QUEST_KEY*: Generate a PAT [here](https://dev.azure.com/msft-skilling/_usersSettings/tokens). WorkItems: Read/Write and Project & Team: Read/Write access are needed.
+     Identity: Read is also needed.
```

Comment:
@adegeo @Rick-Anderson 

Before this fix gets merged, you'll need to update the PAT For the msft-skilling org to add the ability to Read identities.

Reply:
I anticipated more rights would be needed and added them but not Identity. I'll generate new tokens with that.

<img width="178" alt="image" src="https://user-images.githubusercontent.com/3605364/202304515-12222719-8d4a-4a90-b783-199a23216411.png">

Reply:
@BillWagner  do you want to update [Installation and use](https://github.com/dotnet/docs-tools/tree/main/actions/sequester#installation-and-use) ?

Reply:
@BillWagner  edited and added `Identity: Read`.